Gifski/util.swift

```diff
@@ -2417,3 +2418,45 @@ extension URL {
 		try attributes.set("com.apple.metadata:\(key.attributeKey)", value: value)
 	}
 }
+
+extension NSViewController {
+	func push(viewController: NSViewController, completion: (() -> Void)? = nil) {
+		guard let window = view.window else {
+			return
+		}
+
+		let newOrigin = CGPoint(x: window.frame.midX - viewController.view.frame.width / 2.0, y: window.frame.midY - viewController.view.frame.height / 2.0)
+		let newWindowFrame = CGRect(origin: newOrigin, size: viewController.view.frame.size)
+
+		let animation = CABasicAnimation()
```

Comment:
Would be cool to use CASpringAnimation to make the transition even more organic.